leptonai/api/v1/client.py

```diff
@@ -11,6 +11,7 @@
 
 from leptonai.api.util import _get_full_workspace_api_url
 
+
```

Comment:
nit: no need to have two blank lines :) You can do
```
pip install black
black --preview leptonai/api/v1
```
to automatically get link processed.